wiki/Skinning/skin.ini/en.md

```diff
@@ -160,18 +160,19 @@ osu! organises the commands with a heading command. Which may look like this `[G
 - `Name:`
   - Question: What is the name of this skin?
   - Value: text *(skin name)*
-  - Default: *(empty)*
+  - Default: `Unknown`
   - Notes:
-    - At this time, this command is not used.
-    - The osu!client will refer to the folder's name instead when listing.
+    - After exporting the skin, the filename is `<Name>.osk`.
+      - In [osu!(lazer)](/wiki/Client/Release_stream/Lazer), the filename is `<Name> (<Author>).osk`.
+    - The skin selector uses folder names, not the names given by this option.
+      - In [osu!(lazer)](/wiki/Client/Release_stream/Lazer), the skin selector uses the name and author from `skin.ini`, as well as the folder name if it differs from the skin's name.
```

Comment:
Is this really clarifying anything? I don't understand the goal, unless the goal is to confuse people more.

Reply:
the first point is correcting the main mistake of calling it unused (https://github.com/ppy/osu-wiki/issues/11234#issuecomment-2101661228), also just seems like non-obvious info to me because I actually thought it was unused too. the second point is a common misconception that was already noted here so I just reworded that part and added lazer info

im not really seeing the issue, these notes explain what the option does (and doesn't)

Reply:
so it uses the name and author and occasionally the folder name, but in what format? seems missing compared to the other bullet points

pretty sure the goal is to provide an accurate description, as with everything on the wiki, though a screenshot showcasing the usage would be better

Reply:
I thought it would be too verbose but ok I added that. also, peppy was commenting on this whole PR, not just the last line